common/scala/src/main/resources/application.conf

```diff
@@ -344,6 +344,13 @@ whisk {
     #   }
     # }
 
+    activation-store {
```

Comment:
is this stanza required? If not specified is there a default policy (no retry?) assumed?
Since this has no backoff policy, the retry here is really only handling transient failures.

Reply:
May be it is necessary to add retry by default?